src/styles/components/_profile.scss

```
@@ -87,6 +87,11 @@
     height: 1em;
   }
 }
+
+.responsive-px {
+  padding: $responsivePx;
+}

```

```suggestion
.responsive-px {
  padding: 0 $responsivePx;
}
```

Just add 0 here for padding top and bottom, like the one we had earlier.

---

Thank you for checking! I fixed it, and will merge soon :)